packages/hoprd/src/commands/settings.ts

```diff
@@ -70,15 +71,17 @@ export default class Settings extends AbstractCommand {
     return results.join('\n')
   }
 
-  private getState(setting: string, state: GlobalState): string {
+  private getSingleState(setting: string, state: State): string {
     if (this.settings[setting] && this.settings[setting][2]) {
       // Use getter
       return this.settings[setting][2]()
     }
-    return state[setting]
+    return state.settings[setting]
   }
 
-  public async execute(log, query: string, state: GlobalState): Promise<void> {
+  public async execute(log, query: string, { getState }: StateOps): Promise<void> {
```

Comment:
We should be using `setState` somewhere here, tested locally and the setting doesn't update. Unfortunately our tests don't capture this!